eng/pipelines/performance/templates/perf-arm64-build-jobs.yml

```diff
@@ -0,0 +1,77 @@
+parameters:
+  mono: false
+  coreclr: false
+  monoAot: false
+
+jobs:
+  - ${{ if eq(parameters.mono, true) }}:
+    # build mono
+    - template: /eng/pipelines/common/platform-matrix.yml
+      parameters:
+        jobTemplate: /eng/pipelines/common/global-build-job.yml
+        buildConfig: release
+        runtimeFlavor: mono
+        platforms:
+        - linux_arm64
+        jobParameters:
+          nameSuffix: mono
+          buildArgs: -s mono+clr.iltools+clr.hosts+libs+host+packs -c $(_BuildConfig)
```

Comment:
Is `clr.iltools+clr.hosts` required for mono jobs?

Reply:
These arguments are unchanged from what they were before, perhaps it was a copy-paste error back from when these jobs were introduced. I will fix this in this PR as well and make sure it's all good.

Reply:
It's possible that corerun and ilasm are needed if this builds/runs any runtime tests. Otherwise it can probably be dropped.

Reply:
I made the change but it failed, I think the failure is probably because of a bug in our python scripts though having a dependency on it even when it's not being used. I am going to add it back but we can look at removing that dependency in a future PR